src/components/ArticleLabel/Styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+
+import {fonts} from 'styles';
+
+export const Container = styled.div`
+  width: 100%;
+  padding: 15px;
+`;
+
+export const Box = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0px 15px 0px;
+  width: 100%;
+`;
+export const BoxLeft = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+`;
+export const Domain = styled.div`
```

Comment:
This `Domain` is also confusing, since in reality, it's just rendering some content
<img width="484" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/7e629bff-c7d2-410f-a1ee-bb180c8b218c">